packages/drawer/src/Drawer/Drawer.types.ts

```diff
@@ -2,10 +2,29 @@ import React from 'react';
 
 import { DarkModeProps, HTMLElementProps, LgIdProps } from '@leafygreen-ui/lib';
 
+/**
+ * Options to control how the drawer element is displayed
+ * @param Overlay will display a drawer that takes up the full viewport height and elevated above main page content
+ * @param Persistent will display a drawer that takes up the full parent container height and on the same elevation as main page content. It is recommended to wrap a persistent drawer within the `PersistentDrawerLayout` container
```

Comment:
I still think `inline` might be a better descriptor for this behavior

Reply:
> opened alongside the trigger element

That's super fair.

My concern with the name "persistent" is that whether the drawer actually "persists" is not on us, but rather controlled by the consumer.  (Though I may be over-thinking this 😅)

The behavior we control is that the drawer is _beside_ the content, and not above. I'm not opposed to `embedded`